spec/rack/test/uploaded_file_spec.rb

```diff
@@ -26,4 +32,29 @@ def test_file_path
 
     expect(File.extname(uploaded_file.path)).to eq(".txt")
   end
+
+  context "it should call its destructor" do
+    it "calls the destructor" do
+      expect(Rack::Test::UploadedFile).to receive(:actually_finalize).at_least(:once)
+
+      if RUBY_PLATFORM == 'java' && RUBY_VERSION == '2.3.1'
```

Comment:
Current JRuby 's Ruby version is 2.3.3. `jruby-head` is failed because of the reason.
I do not want to write actual version number in the test code.
Can you remove `&& RUBY_VERSION == '2.3.1'` from the code?